website/src/components/examples.js

```diff
@@ -32,15 +26,22 @@ export default class Examples extends Component {
     window.onresize = null;
   }
 
+  _getSize() {
+    const {innerWidth, innerHeight} = window;
+
+    return {
+      width: innerWidth >= 576 ? innerWidth - WIDTH_OFFSET : innerWidth,
```

Comment:
We have a media selector in css for mobile. Not sure if there are better ways to manage this variable other than hard coding it.